### DISCUSSION TITLE
using rotary encoder with quick develop

### DISCUSSION THREAD

**Discussion by cstsang** [rsjaffe/MIDI2LR#533] [Ideas]
Is there a way to setup a rotary encoder (ex: cc1) to quick develop exposure +1/3  and exposure -1/3?  I know the quick develop is not a slider, but is it possible?  so turn right is exposure +1/3 and turn left is exposure -1/3?

thanks!

**Comment by rsjaffe** [comment]
Currently, no. I could look into something like that for the next release. Note however, that it could be difficult to control the effect, as Lightroom could be laggy when applying a quick develop adjustment to lots of selected photos.

**Comment by cstsang** [comment]
I would be happy if it works just for the 1 photo I have selected.  Won't
even be greedy, may be happy if it just works for the exposure 1/3
buttons.  thanks!

**Comment by rsjaffe** [comment]
If you want to test it out, extract the three lua files from [ClientUtilities.zip](https://github.com/rsjaffe/MIDI2LR/files/7566057/ClientUtilities.zip), place them in your MIDI2LR.lrplugin directory, delete the _MenuTrans.xml_ file that is found in _%AppData%\MIDI2LR_ (Windows) or _~/Library/Application Support/MIDI2LR_ (MacOS) and start up Lightroom.

**Reply by cstsang** [reply]
Just trying it now.  Works GREAT on the exposure, black, highlight, shadow, tint, and temp!  Will try other quick develop settings and let you know if I run into any problems.  This is a game changer for me! 

thank you so much.

**Comment by rsjaffe** [comment]
It's in version 5.2

**Comment by cstsang** [comment]
works great!

thank you so much!
